### DISCUSSION TITLE
Referencing an equation

### DISCUSSION THREAD

**Discussion by spring-haru** [executablebooks/meta#364] [Q&A] [answered]
I referenced an equation with the following:

```
$$
y= x+1
$$ (eq:var(a))

Equation {eq}`var(a)` is great!
```

but it generates a warning

```
WARNING: equation not found: var(a)
```

and the result is not as expected. The same result for `[](var(a))`.

On the other, if `()` is removed, then it works fine.

```
$$
y= x+1
$$ (eq:var_a)

Equation {eq}`var_a` is great!
```

Is it an expected result?

**Comment by choldgraf** [comment, marked as answer]
Yeah I suspect that it is expected, the parentheses likely mess up the parsing regex. Maybe you could escape the interior parentheses with backslashes?
